src/book.js

```diff
@@ -763,7 +842,7 @@ export class Book {
      * PDF builder base function for Classic (non-Wacky) layouts. Called by [createoutputfiles]
      *
      * @param config - object w/ the following parameters:
-     *    - pageIndexes : a nested list of original source document page numbers ordered for layout. ( [0] for duplex & front, [1] for backs -- value is null if no aggregate printing enabled). Ex: [[4, 3, 2, 5, 6, 1, 0, 7]]
+     *    - pageIndexDetails : a nested list of original source document page numbers ordered for layout. ( [0] for duplex & front, [1] for backs -- value is null if no aggregate printing enabled). Ex: [[4, 3, 2, 5, 6, 1, 0, 7]]
```

Comment:
This is a list of pageInfo objects, right? Not just a list of integers any more?

Reply:
🙏 yes, thank you good catch